test/functional/system/blocks/chain/delete_last_block_db_trs.js

```diff
@@ -0,0 +1,403 @@
+/* eslint-disable mocha/no-skipped-tests */
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const expect = require('chai').expect;
+const lisk = require('lisk-js');
+const accountFixtures = require('../../../../fixtures/accounts');
+const randomUtil = require('../../../../common/utils/random');
+const localCommon = require('../../common');
+const genesisBlock = require('../../../../data/genesis_block.json');
+
+describe('system test (blocks) - chain/popLastBlock', () => {
+	const transferAmount = 100000000 * 100;
+	let library;
+	let db;
+
+	localCommon.beforeBlock('system_blocks_chain_pop_last_block', lib => {
+		library = lib;
+		db = library.db;
+	});
+
+	afterEach(done => {
+		db
+			.task(t => {
+				return t.batch([
+					db.none('DELETE FROM blocks WHERE "height" > 1;'),
+					db.none('DELETE FROM forks_stat;'),
+					db.none('UPDATE mem_accounts SET "producedBlocks" = 0'),
+				]);
+			})
+			.then(() => {
+				library.modules.blocks.lastBlock.set(genesisBlock);
+				done();
+			})
+			.catch(err => {
+				__testContext.debug(err.stack);
+				done();
+			});
+	});
+
+	let block;
+	let blockAccount1;
+	let fundTrsForAccount1;
+
+	beforeEach('send funds to accounts', done => {
+		blockAccount1 = randomUtil.account();
+		fundTrsForAccount1 = lisk.transaction.createTransaction(
+			blockAccount1.address,
+			transferAmount,
+			accountFixtures.genesis.password
+		);
+
+		fundTrsForAccount1.senderId = accountFixtures.genesis.address;
+
+		localCommon.createValidBlock(library, [fundTrsForAccount1], (err, b) => {
+			expect(err).to.not.exist;
+			block = b;
+			library.modules.blocks.chain.applyBlock(block, true, done);
+		});
+	});
+
+	describe('popLastBlock', () => {
+		let processExit;
+
+		beforeEach(done => {
+			processExit = sinonSandbox.stub(process, 'exit');
+			return done();
+		});
+
+		afterEach(done => {
+			processExit.restore();
+			done();
+		});
+
+		describe('when it fails', () => {
```

Comment:
Please remove empty describe block - merge them with the bottom ones. Here and in the similar examples. Follow the pattern:
```
describe('when loadBlockSecondLastBlockStep fails`)
```
instead of 
```
describe('when it fails`)
   describe('when loadBlockSecondLastBlockStep fails`)
```